backend/dal/Services/Interfaces/IResearchService.cs

```diff
@@ -0,0 +1,10 @@
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Models;
+
+namespace Pims.Dal.Services
+{
+    public interface IResearchService
+    {
+        Paged<PimsResearchFile> GetPage(ResearchFilter filter);
+    }
+}
```

Comment:
Do we have a use for this interface - given that we already have a **IResearchFileService** above? seems like a duplicate

Reply:
will remove